    /**
     * See bug #61700
     */
    @Test
    void testWorkbookForceFormulaRecalculation() throws Exception {
        Workbook workbook = _testDataProvider.createWorkbook();
        workbook.createSheet().createRow(0).createCell(0).setCellFormula("B1+C1");
        workbook.getCreationHelper().createFormulaEvaluator().evaluateAll();

        assertFalse(workbook.getForceFormulaRecalculation());
        workbook.setForceFormulaRecalculation(true);
        assertTrue(workbook.getForceFormulaRecalculation());

        Workbook wbBack = _testDataProvider.writeOutAndReadBack(workbook);
        assertTrue(wbBack.getForceFormulaRecalculation());
        wbBack.setForceFormulaRecalculation(false);
        assertFalse(wbBack.getForceFormulaRecalculation());

        Workbook wbBack2 = _testDataProvider.writeOutAndReadBack(wbBack);
        assertFalse(wbBack2.getForceFormulaRecalculation());

        workbook.close();
        wbBack.close();
        wbBack2.close();
    }

